src/shared/awsContext.ts

```diff
@@ -5,6 +5,12 @@
 
 import * as vscode from 'vscode'
 
+export interface AwsContextCredentials {
+    readonly credentials: AWS.Credentials
+    readonly credentialsId: string
```

Comment:
I'd recommend leaving this as profileName, it's still referred to as such everywhere else in the codebase.

Reply:
Over the course of bringing on a new credentials manager, we should transform code away from the concept of "profile" over to "credentials". Profile is a subset of Credentials, its just that its the only portion we've been working with so far. (The term 'profile' will likely be used with any credentials coming from the credentials/config shared ini files as part of the credentialsId, much like the JetBrains toolkit does)